libr/core/cmd_info.c

```diff
@@ -253,7 +253,7 @@ static int cmd_info(void *data, const char *input) {
 	int is_array = 0;
 	Sdb *db;
 
-	for (i = 0; input[i] && i < 2; i++) {
+	for (i = 0; input[i]; i++) {
```

Comment:
imho `irszM*` will work since the patch is not assuming '*' appears in the first 2 chars, current code is assuming this. I remove the check for `i < 2` and now it will keep checking the mode chars till the end, or find one and break. I will update the patch to only look at the last char, though.